functions.php

```diff
@@ -324,17 +323,12 @@ function colormag_content_width() {
 			if ( 'no_sidebar_full_width' === $colormag_default_post_layout ) {
 				$content_width = 1140; /* pixels */
 			}
-		} else {
-			if ( 'no_sidebar_full_width' === $colormag_default_sidebar_layout ) {
+		} elseif ( 'no_sidebar_full_width' === $colormag_default_sidebar_layout ) {
 				$content_width = 1140; /* pixels */
-			}
 		}
-	} else {
-		if ( 'no_sidebar_full_width' === $layout_meta ) {
+	} elseif ( 'no_sidebar_full_width' === $layout_meta ) {
```

Comment:
Are these changes intended? @jenisha00

Reply:
No, these changes are not intended but occurred due to formatting. @asheshmagar

Reply:
Then revert them formatting tries to convert the lonely if inside else to elseif but in some condition lonely if inside else if required